### PR TITLE
 feat: extend deal info with tasks and resources

### PR DESCRIPTION
**Pull request by sshaman1101**

### CODE REVIEW

**Comment by sshaman1101** [comment]
@antmat @3Hren PTAL

File: cmd/cli/commands/printers.go [hunk 1]
```diff
@@ -311,8 +313,41 @@ func printDealInfo(cmd *cobra.Command, deal *pb.Deal) {
 		if lastBill.Unix() > 0 {
 			cmd.Printf("Last bill:    %s\r\n", lastBill.Format(time.RFC3339))
 		}
+
+		if info.Resources != nil {
+			cmd.Println("Resources:")
+
+			cpuCores := float64(info.Resources.CPU.CorePercents) / 100.0
```
Comment (antmat) [inline cmd/cli/commands/printers.go]: use getters, Luke!

File: cmd/cli/commands/printers.go [hunk 2]
```diff
@@ -311,8 +313,41 @@ func printDealInfo(cmd *cobra.Command, deal *pb.Deal) {
...
+			ram := info.Resources.RAM.Size.Unwrap().HumanReadable()
```
Comment (antmat) [inline cmd/cli/commands/printers.go]: May the getters be with you!

File: insonmnia/miner/server.go [hunk 3]
```diff
@@ -907,14 +895,44 @@ func (m *Miner) GetDealInfo(dealID *pb.BigInt) (*pb.DealInfoReply, error) {
...
 	}
-	//TODO: why do we need BidOrder and AskOrder inside this reply?
-	//TODO: Tasks info
+
+	ask, err := m.salesman.AskPlanByDeal(dealID)
+	if err != nil {
+		return nil, err
+	}
+
+	m.mu.Lock()
```
Comment (antmat) [inline insonmnia/miner/server.go]: moveforward

**Pull request merged**